lib/srv/db/common/errors.go

```diff
@@ -236,3 +236,14 @@ agent's service principal. See: https://goteleport.com/docs/database-access/guid
 		return trace.Wrap(err)
 	}
 }
+
+// IsUnrecognizedAWSEngineNameError checks if the err is non-nil and came from using an engine filter that the
+// AWS region does not recognize.
+func IsUnrecognizedAWSEngineNameError(err error) (res bool) {
```

Comment:
I debugged to see the type of error.

The err is an `awserr.requestError`, it's not wrapping an `InvalidParameterValue` error. The "InvalidParamterValue" is a part of the error's string content. Since this fix is specific to unrecognized engine names, I think I'd prefer to just check for the "Unrecognized engine name" substring in the error message